utils/protocol-types-generator/index.js

```diff
@@ -4,6 +4,7 @@ const puppeteer = require('../..');
 module.exports = puppeteer.launch({
   pipe: false,
   executablePath: process.env.CHROME,
+  args: [ '--no-sandbox' ],
```

Comment:
ok, there are more into this: there's not only protocol types, but also our linter, and all of our tests. You'll probably want to add this flag in all these places.

We had `--no-sandbox` everywhere some time ago. But since we figured the proper sandboxing setup and documented it in our troubleshooting guide, we dropped them all to actually validate that things work.

Adding `--no-sandbox` makes our CI story weaker - let's not do it.

Reply:
Ok, so maybe the actual fix is to not generate the protocol types when installing puppeteer via npm/yarn ?